controllers/replication/secret_controller_test.go

```diff
@@ -253,6 +265,8 @@ func generateSecrets(namespace string) []*corev1.Secret {
 				Name:      "test-secret-first",
 				Labels: map[string]string{
 					"secret-controller": "test",
+					"cluster-namespace": namespace,
+					"cluster-name":      "k8ssandra",
```

Comment:
Isn't it better to use the available constants here?

Reply:
Yeah, I'll update it.